### PR TITLE
Fixes #5582; Removing Black Squad

### LINKED ISSUE #5582: Black squad scrapped; Update black squad test marker with respective squads
The black squad is scrapped. So, it is important to update markers in ocs-ci with the respective squads.

### PR DESCRIPTION
**Pull request by keesturam**
Signed-off-by: Krishnaram Karthick Ramdoss <kramdoss@redhat.com>

### CODE REVIEW

**Review by vavuthu — APPROVED** [review]

**Review by vavuthu — COMMENTED** [review]
please remove from https://github.com/red-hat-storage/ocs-ci/blob/dffe18d502007544da8e88d0c7c3214a302273dd/pytest.ini#L53

and

https://github.com/red-hat-storage/ocs-ci/blob/f57e12c4049c409570a9103b8807c82e5630e1bf/ocs_ci/framework/pytest_customization/marks.py#L371

**Review by ebenahar — DISMISSED** [review]

File: tests/ui/test_validation_ui.py [hunk 1]
```diff
@@ -20,6 +21,7 @@ class TestUserInterfaceValidation(object):
 
     @ui
     @tier1
+    @blue_squad
```
Comment (fbalak) [inline tests/ui/test_validation_ui.py:24]: This is not a blue squad test case.

File: tests/ui/test_validation_ui.py [hunk 2]
```diff
@@ -40,6 +42,7 @@ def test_dashboard_validation_ui(self, setup_ui):
     @tier1
     @skipif_ui_not_support("validation")
     @skipif_ocs_version("<4.9")
+    @blue_squad
```
Comment (fbalak) [inline tests/ui/test_validation_ui.py:45]: This is not a blue squad test case.

**Review by fbalak — CHANGES_REQUESTED** [review]
Blue squad is focused on monitoring testing, not on UI validations.

File: tests/ui/test_validation_ui.py (hunk 2 above)
Comment (keesturam) [reply tests/ui/test_validation_ui.py:45]: @ebenahar any thoughts on which squad could be a better fit? 
This is a basic test that checks if the dashboard works. so it is not blue squad as per our definitions. But, it doesn't fall under any other category as well. 
Maybe purple squad?

**Review by am-agrawa — APPROVED** [review]
LGTM

**Review by jilju — APPROVED** [review]

File: tests/ui/test_validation_ui.py (hunk 2 above)
Comment (ebenahar) [reply tests/ui/test_validation_ui.py:45]: indeed no squad has this area by definition. 
@keesturam can you keep it under black for now (even though we would like to deprecate this squad), until we define which squad will take this area

**Review by fbalak — APPROVED** [review]

**Review by am-agrawa — APPROVED** [review]

**Review by jilju — APPROVED** [review]

**Pull request merged**